configuration/rules-dsl.md

```diff
@@ -450,9 +452,6 @@ val location = new PointType(new DecimalType(50.12345), new DecimalType(10.12345
 // Creation from String; ATTENTION: do not add space after comma
 val PointType home = new PointType("12.121212,123.123123")
 
-// Saving to an Item
-Device_Coordinates.postUpdate(location)
```

Comment:
Why did you remove this example?

Reply:
I removed all the postUpdate and sendCommand examples as this is handled in the section above in more details. No strong feelings either way, it appeared redundant and superfluous to me at this point, but happy to put it back in, as it won't do harm either

Reply:
Ah. I see. Nope sounds good!